rpc/errors.go

```diff
@@ -61,12 +61,14 @@ const (
 	errcodeDefault                  = -32000
 	errcodeNotificationsUnsupported = -32001
 	errcodeTimeout                  = -32002
+	errcodeResponseTooLarge         = -32003
```

Comment:
This condition could have its own error code!

Reply:
yes? `-32003` is own error code

Reply:
Sorry, you're right, I didn't see the difference with `-32603` below.

Reply:
yea, these 2 [codes](https://github.com/ethereum/go-ethereum/blob/667a4088d22ada3878a500deab16dd665824a1d8/rpc/errors.go#L65-L66) are the same.